File: FrontEnd/Day4/DataTypes/Exercises/fizzBuzz.js
let fizzBuzz = () => {
    let input = parseInt(window.prompt("Enter a number between 1 & 100"));
    
    if(isNaN(input)){
        alert("Please enter a NUMBER");
        fizzBuzz();
    } else if(input < 1){
        alert("Number too low");
        fizzBuzz();
    } else if(input > 100){
        alert("Number too high");
        fizzBuzz();
    } else{
        if(input % 3 === 0 && input % 5 === 0){
            alert("FizzBuzz");
        } else if(input % 3 === 0){
            alert("Fizz");
        } else if(input % 5 === 0){
            alert("Buzz");
        } else{
            alert(`Number is: ${input}`);
        }
    } 
}

fizzBuzz();

File: BackEnd/Day1/fizzbuzz.py
while True:
    try:
        user_input = int(input("Please enter a number: "))
        break
    except ValueError:
        print("bruh a number cmon man")

if (user_input % 3 == 0 and user_input % 5 == 0):
    print("FizzBuzz")
elif user_input % 3 == 0:
    print("Fizz")
elif user_input % 5 == 0:
    print("Buzz")
else:
    print(user_input)

